DEVELOPER.md

```diff
@@ -55,17 +90,33 @@ git remote add upstream https://github.com/MODFLOW-USGS/modflow6.git
 
 ## Building
 
-To build MODFLOW 6 run:
+You can build modflow with Visual Studio Code tasks as described [here](.vscode/README.md).
+For the more general instructions, continue to read this section.
+
+First configure the build directory:
 
 ```shell
-# Go to the pymake directory
-cd modflow6/pymake
+# bash (linux and macOS)
+meson builddir --prefix=$(pwd) --libdir=bin
 
-# run the modflow6 pymake build script
-python makebin.py -mc ../src/ ../bin/mf6
+# cmd (windows)
+meson builddir --prefix=%CD% --libdir=bin
 ```
 
-* Results are put in the bin folder.
+For the following compilations, only this command has to be executed:
+
+```shell
+meson compile -C builddir
+```
+
+In order to run the tests the binaries have to be installed by executing:
+
+```shell
+meson install -C builddir
+```
+
+The binaries can then be found in the `bin` folder.
+
 
 ## Running Tests Locally
```

Comment:
Probably we want to remove the last mention of Travis and put Github Actions there